### ISSUE TITLE
Whitespace

### ISSUE THREAD

**Issue by ramonsmits** [EliasOenal/multimon-ng#222]
@EliasOenal I'm considering some small refactorings to unify and cleanup the whitespace in the flex decoder. The file is a little messy. Would you accept a PR that fixes whitespace indentation and tried to keep the code pretty much identical as is?

Alternatively we could reformat the whole file with a specific code formatter. That would be great for future changes and auto formatting but it would make it harder to compare changes with prior versions.

**Comment by EliasOenal** [comment]
Yes, please make a separate commit for just the whitespace changes and as needed another one for non-whitespace refactoring. As of now the project overall does not have consistent formatting, but I'd be happy to discuss options for this. Ideally we'd use something that can be run from GitHub CI. Maybe we could add two targets to cmake to verify formatting (used for CI) and to apply formatting. I know `clang-format` is popular, do you have any recommendation?
